sdk/python/feast/repo_config.py

```diff
@@ -56,8 +68,10 @@ class RegistryConfig(FeastBaseModel):
 class RepoConfig(FeastBaseModel):
     """ Repo config. Typically loaded from `feature_store.yaml` """
 
-    registry: Union[StrictStr, RegistryConfig] = "data/registry.db"
-    """ str: Path to metadata store. Can be a local path, or remote object storage path, e.g. a GCS URI """
+    registry: Union[StrictStr, RegistryConfig] = RegistryConfig()
```

Comment:
Is there a reason to stop using `"data/registry.db"` as the default configuration path and create a whole object?

Reply:
I was assuming we wanted to change the default to use the nested config for the additional options, but since we can just accommodate both moving forward, I'll change the default to the simple string